### ISSUE TITLE
Remove firefox directory

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat-reporter-extensions#35]
We moved to a WebExtension, which is shipping now, so we can just get rid of the old one. And then we can rename firefox-webext to just firefox.

**Comment by miketaylr** [comment]
to be clear: remove https://github.com/webcompat/webcompat-reporter-extensions/tree/master/firefox

rename https://github.com/webcompat/webcompat-reporter-extensions/tree/master/firefox-webext to just `firefox`

**Comment by miketaylr** [comment]
(note: this is super low priority, we can wait on it -- to not cause conflicts for people who are hacking on things right now)

**Comment by AkhilHector** [comment]
i got this @miketaylr

**Comment by miketaylr** [comment]
Let's hold off a few days @AkhilHector or maybe, just as part one remove `/firefox/`. Otherwise @adamopenweb and I are gonna end up with some merge conflicts.

**Comment by miketaylr** [comment]
(and then we can rename in a follow up bug)

**Comment by AkhilHector** [comment]
oh al right no issue man. I think we will hold off for a few days @miketaylr

**Comment by miketaylr** [comment]
ya sgtm

**Comment by miketaylr** [comment]
@AkhilHector should be fine to proceed here!

**Comment by AkhilHector** [comment]
Ok @miketaylr
